### PR TITLE
Home Page UI -Deploy Preview

### PR DESCRIPTION
**Pull request by ultronjs**
Added the Home Page UI and Link to the components

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -4,56 +4,56 @@ import { Link } from "react-router-dom";
...
                 My Cart
               </Link>
             </li>
-            <li class="px-x-small">
-              <Link class="link_wrapper" to="/wishlist">
+            <li className="px-x-small">
+              <Link className="link_wrapper" to="/wishlist">
                 My Waitlist
               </Link>
             </li>
```
Comment (mariyasada) [inline src/components/Footer.jsx:31]: fix typo waitlist to wishlist

File: src/components/GamesFromBrands.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+function GamesFromBrands() {
+  const brands = [
+    { name: "Sony", linkTo: "/products", imgSrc: SonyImage },
+    { name: "RockStar", linkTo: "/products", imgSrc: RockStarImage },
+    { name: "EA", linkTo: "/products", imgSrc: EAImage },
+    { name: "Warner", linkTo: "/products", imgSrc: WarnerImage },
+    { name: "Marvel", linkTo: "/products", imgSrc: MarvelImage },
+    { name: "Ubisoft", linkTo: "/products", imgSrc: UbisoftImage },
+  ];
+  return (
```
Comment (mariyasada) [inline src/components/GamesFromBrands.jsx:15]: liked that you created array of objects

**Review by mariyasada — APPROVED** [review]
good to go

File: src/components/Footer.jsx (hunk 1 above)
Comment (ultronjs) [reply src/components/Footer.jsx:31]: Fixed.Thanks

**Review by psychoactiv — APPROVED** [review]
Looks clean, good to merge

**Comment by ultronjs** [comment]
Merging after 2 reviews

**Pull request merged**